src/webui/utils/login.js

```diff
@@ -0,0 +1,78 @@
+import isString from 'lodash/isString';
+import isNumber from 'lodash/isNumber';
+import size from 'lodash/size';
+import {Base64} from 'js-base64';
+import API from './api';
+import {HEADERS} from '../../lib/constants';
+
+export function isTokenExpire(token) {
+    if (!isString(token)) {
+        return true;
+    }
+
+    let payload = token.split('.')[1];
```

Comment:
You might try here http://exploringjs.com/es6/ch_destructuring.html#_array-destructuring